packages/scoped-custom-element-registry/test/Element.test.html.js

```diff
@@ -55,4 +78,30 @@ describe('Element', () => {
       });
     });
   });
+
+  describe('insertAdjacentHTML', () => {
+    it('should upgrade an element defined in the custom registry', () => {
+      const {tagName, CustomElementClass} = getTestElement();
+      const registry = new CustomElementRegistry();
+      const shadowRoot = getShadowRoot(registry);
+      const $el = getHTML('<div></div>', shadowRoot);
```

Comment:
why the `$` prefix?

Reply:
Just copy/paste from the existing style of the tests. We can consider a separate task to normalize this repo the style used in the rest of the monorepo.